mne/utils.py

```diff
@@ -353,7 +365,7 @@ def __del__(self):
 
 
 def estimate_rank(data, tol='auto', return_singular=False,
-                  norm=True, copy=True):
+                  norm=True, copy=None):
```

Comment:
This method defaulting to in-place really does not seem like least-surprise to me.

Not that I know a better alternative.